src/auth/schemas/user.schema.ts

```diff
@@ -0,0 +1,33 @@
+import { Optional } from "@nestjs/common";
+import { Schema, Prop, SchemaFactory } from "@nestjs/mongoose";
+import { Document } from "mongoose";
+
+@Schema({ timestamps: true })
+export class User extends Document {
```

Comment:
we need an objectId as primary key

Reply:
Normally, MongoDB will add a field called "_id" whose type is ObjectID, and this field("_id") will be reserved to be the primary key if we did not set our own id as "_id". I noticed you left same comment at other devs' pull request.  Maybe this is the answer you want to get. Please remind me if I said anything wrong.

Reference link: 
https://www.mongodb.com/docs/manual/core/document/
https://www.mongodb.com/docs/manual/reference/glossary/
![image](https://user-images.githubusercontent.com/50484579/175930001-bb174387-d5d3-41e5-afc0-5f7bb4731bb7.png)
![image](https://user-images.githubusercontent.com/50484579/175930066-e0631ee1-366b-485c-8d91-9bb08f49f4b0.png)